index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./css/style.css">
+  <title>Portfolio</title>
+</head>
+<body>
+  <main>
+    <div class="background-image"><img src="./images/Illustration.png"/ alt="Vector image"></div>
+    <header class="header-container">
+      <div class="header-title-container"><h2 class="header-title">John Doe</h2></div>
+      <div class="menu-icon-container"><a class="button header-menu-icon" href="#"><img src="./images/Menu.svg" alt="Menu icon"/></a></div>
+    </header>
+    <section class="headline-container">
+      <h1 class="headline-title">Hey There.<br>I’m Alex</h1>
+      <h2 class="headline-subtitle">I’am a Software Developer</h2>
```

Comment:
Please consider wrapping this part of the headline in a `p` tag instead of an `h2` tag, since you already have an `h1` tag in this `section`. For the page to be semantically correct it would not be best practice to have another headline within it. This will also make it easy for users using web accessibility tools like screen readers to interpret the web page correctly.